FORKING.md

```diff
@@ -0,0 +1,85 @@
+# Die FSI-Seite weggabeln
+Diese Anleitung erklärt, wie man die FSI-Homepage bearbeitet. Die Voraussetzungen dafür sind:
+- Ein GitHub-Account,
+- auf Windows die Git-Shell
+- auf Linux bzw. Mac OS ein installiertes git sowie ein Terminal
+- grundsätzliches Verständnis für Git.
+
+<!-- Für die Git-Klugscheißer: Diese Anleitung orientiert sich an den offiziellen GitHub-Anleitungen https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork und https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork -->
+
+## Den Spaß weggabeln :fork_and_knife:
+Damit du die Homepage bearbeiten kannst, musst du dir zunächst auf deinem GitHub-Account eine Kopie ("Fork") der Fachschaftsseite erstellen.
+Ruf dazu zunächst das [**Repository**](https://github.com/fsi-tue/cyberauftritt) auf. Klicke oben rechts auf "Fork". Du besitzt nun eine Kopie der Fachschaftswebseite. Klicke (auf der Seite deiner Kopie, die sich selbst öffnen sollte) auf den grünen Button "Clone or Download".
+
+
+### Mit GitHub Desktop
+> :information_source: Noch nicht ausprobiert. Freiwillige vor!
+
+### Mit SSH
+> Tipp: Wenn du nicht weißt, was SSH-Keys sind, gehe zum Unterpunkt "mit HTTPS". Begib dich direkt dorthin, gehe nicht über Los, ziehe keine 4000 DM ein.
+
+Kopiere die Zeile, die im Textfeld bei "Clone with SSH" steht. Öffne ein Terminal, gehe in das Verzeichnis in dem ein Unterordner für die Fachschaftsseite erstellt werden soll. Tippe dann im Terminal: `git clone <Strg-Shift-V>`
+Es sollte das hier entstehen:
+```bash
+git clone git@github.com:deinusername/cyberauftritt.git
+```
+Führe den Befehl aus. Der Ordner "cyberauftritt" entsteht.
+### Mit HTTPS
+Klicke auf "Use HTTPS". Kopiere die Zeile, die im Textfeld bei "Clone with HTTPS" steht. Öffne ein Terminal, gehe in das Verzeichnis in dem ein Unterordner für die Fachschaftsseite erstellt werden soll. Tippe dann im Terminal: `git clone <Strg-Shift-V>`
+Es sollte das hier entstehen:
+```bash
+git clone git@github.com:deinusername/cyberauftritt.git
+```
+Führe den Befehl aus. Du wirst nach GitHub-Benutzernamen und dem dazugehörigen Passwort gefragt.
+Der Ordner "cyberauftritt" entsteht.
+
+## Original-Repository als Referenz hinzufügen
+
+Je nach dem wie viel auf der Seite gerade geändert wird, musst du dafür sorgen, dass deine Kopie mit dem Original auf einem Stand bleibt. Hierzu musst du einen sog. "Upstream" hinzufügen.
+- `cd /[…]/cyberauftritt`
```

Comment:
Ist möglicherweise verwirrend (wegen dem `/[...]`), ich habe aber auch nichts dagegen. Alternativ evtl. einfach die Anweisung in das lokale Git repo für `cyberauftritt` zu navigieren.

Reply:
Stimmt. Habs jetzt mal so:
- `cd <Verzeichnis zu deinem 'cyberauftritt'-Fork>`